support-frontend/app/controllers/RedemptionsController.scala

```diff
@@ -0,0 +1,35 @@
+package controllers
+
+import actions.CustomActionBuilders
+import com.gu.support.config.{Stage, TouchPointEnvironments}
+import com.gu.support.redemption.{GetCodeStatus, RedemptionCode, RedemptionTable}
+import play.api.mvc.{AbstractController, Action, AnyContent, ControllerComponents}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class RedemptionsController(
```

Comment:
I have a version of this in my corporate subs part 2 PR so there'll be a bit of a merge to do. We could keep this in a separate controller maybe but would need to get the naming right

Reply:
one way to do it might be to just merge both, and then do a separate pr to connect them together